### PR TITLE
Updated to handle the string `null` in the response.

### PR DESCRIPTION
**Pull request by martyphee**
Ravelin sends us `null` in the response if there wasn't an update to the customer record.  This change is to handle that.

### CODE REVIEW

**Comment by martyphee** [comment]
@mathildathompson @jphastings Can i get a review?

File: spec/ravelin/client_spec.rb [hunk 1]
```diff
@@ -165,6 +165,26 @@
...
+    let(:event) { double('event', name: :ping, serializable_hash: {}) }
+    let(:client) { described_class.new(api_key: 'abc') }
+
+    before do
+      allow(Ravelin::Event).to receive(:new).and_return(event)
+      stub_request(:post, 'https://api.ravelin.com/v2/ping').
+        and_return(status: status_code, body: "null")
+    end
+
+    context 'HTTP status 400' do
```
Comment (mathildathompson) [inline spec/ravelin/client_spec.rb]: 'HTTP status 200' ?

File: spec/ravelin/client_spec.rb [hunk 2]
```diff
@@ -165,6 +165,26 @@
...
+      let(:status_code) { 200 }
+
+      it "successfull" do
```
Comment (mathildathompson) [inline spec/ravelin/client_spec.rb]: successful

**Comment by mathildathompson** [comment]
Wow that was a quick update! Dont have a huge amount of context about when this call is being make to Ravelin, but looks fine to me :-)

**Comment by martyphee** [comment]
We're getting a ton of errors in resque because of this.

image

I've talked to Ravelin and they're not planning on fixing this anytime soon.

Thank you.

**Pull request merged**